qiskit/synthesis/linear/cnot_synth.py

```diff
@@ -29,19 +29,20 @@ def synth_cnot_count_full_pmh(state, section_size=2):
 
     This function is an implementation of the Patel, Markov and Hayes algorithm from [1]
     for optimal synthesis of linear reversible circuits for all-to-all architecture,
-    as specified by an n x n matrix.
+    as specified by an :math:`n \\times n` matrix.
```

Comment:
Nothing to change, just a side note: by turning the docstring into a raw string you can use `\times` without the double backslash 🙂